pom.xml

```diff
@@ -63,6 +63,13 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-stack-depchain</artifactId>
+        <version>${vertx.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
It has been added because we have transitive dependency of vertx-stack-depchain with different verx version in classpath?

Reply:
i would say, it is possible that dependency versions could be different in future, but currently versions are the same